Safe-Places-Server.md

```diff
@@ -0,0 +1,222 @@
+# Safe Places Server
```

Comment:
Need some verbiage somewhere describing how to get the authority added to the master list - e.g. healthcare-authorities.yaml.  I'm thinking a separate doc that describes that process (how to update the file, the review process, etc), with just a simple description and link in this doc.